_query-dsl/specialized/script-score.md

```diff
@@ -328,5 +328,52 @@ GET blogs/_search
 ```
 {% include copy-curl.html %}
 
+### Term frequency functions
+
+Term frequency functions expose term-level statistics in a script score context. You can use these statistics to implement custom information retrieval and ranking algorithms like query-time multiplicative or additive score boosting by popularity. To apply a term frequency function, call one of the following Painless methods:
```

Comment:
There's `script context` https://github.com/noCharger/OpenSearch/blob/main/server/src/main/java/org/opensearch/script/ScriptContext.java and each script has their own context instance https://github.com/noCharger/OpenSearch/blob/main/server/src/main/java/org/opensearch/script/ScoreScript.java#L287. I would say expose them in score script `source`.